Sources/Sentry/SentrySDK.m

```diff
@@ -43,6 +44,16 @@ + (void)setCurrentHub:(SentryHub *)hub
     }
 }
 
++ (BOOL)crashedLastRunCalled
+{
+    return crashedLastRunCalled;
+}
+
++ (void)setCrashedLastRunCalled:(BOOL)value
+{
+    crashedLastRunCalled = value;
+}
```

Comment:
M: do we need getters and setters for this? ideally, it'd be a private API only called by Sentry itself

Reply:
This is a Sentry private API. It is made visible to other classes inside Sentry via [`SentrySDK+Private.h`](https://github.com/getsentry/sentry-cocoa/blob/feat/on-crashed-last-run/Sources/Sentry/include/SentrySDK+Private.h#L14). Both methods are not in the `SentrySDK.h` header, so our users won't see them.